src/infinistore.cpp

```diff
@@ -160,6 +160,11 @@ Client::~Client() {
         tcp_recv_buffer_ = NULL;
     }
 
+    if (comp_channel_) {
```

Comment:
1. use ibv_destroy_comp_channel instead of free
2. remove comp fd listen before destroy
3. quick hint: no this->XX needed. XX_. All suffix’s underscore notes the class variable. check the google C++ style

Reply:
1. added
2. poll_handle_ is already released on line 130
3. not very clean, you mean I should use comp_channel instead of comp_channel_?

Reply:
no, use comp_channel_ directly instead of this->comp_channel_. Other class variables follow the same rule

Reply:
okay, updated